docs/scripting/scripts.lib/uniq.mdx

```diff
@@ -25,8 +27,9 @@ Returns an array.
 ```js
 function(context, args) {
   const l = #fs.scripts.lib();
-  const my_array = ["apples", "bananas", "oranges", "apples", "limes", "oranges"];
+  const my_array = ["apples", "apples", "bananas", "oranges", "apples", "limes"];
 
-  return l.uniq(my_array);
+  const uniq_array = l.uniq(my_array); // => ["apples", "bananas", "oranges", "apples", "limes"]
+  return uniq_array;
```

Comment:
```suggestion
  return l.uniq(my_array); // => ["apples", "bananas", "oranges", "apples", "limes"]
```

Reply:
Was unsure because it doesn't print that without `#D`, but it does look nicer :D